modules/@apostrophecms/translation/ui/apos/components/AposTranslationMeta.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+  <AposTranslationIndicator
+    v-if="isShowed"
+    label="apostrophe:automaticTranslationFieldMeta"
+  />
+</template>
+
+<script>
+export default {
+  name: 'AposTranslationMeta',
+  props: {
+    meta: {
+      type: Object,
+      required: true
+    },
+    metaRaw: {
+      type: Object,
+      required: true
+    },
+    field: {
+      type: Object,
+      required: true
+    },
+    items: {
+      type: Array,
+      default() {
+        return [];
+      }
+    }
+  },
+  data() {
+    return {
+      isShowed: this.meta.data?.changed
+    };
+  }
+};
+</script>
```

Comment:
New component handling meta logic for translation. Uses already existing `AposTranslationIndicator.vue`.
Some props are unused here and could be removed. Wasn't sure if we are going to need them soon so I kept them.